developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -18,15 +20,17 @@ def submit_transaction_helper(self, account: Account, payload: Dict[str, Any]):
 #:!:>section_1
     def create_collection(self, account: Account, name: str, description, uri: str):
         """Creates a new collection within the specified account"""
-
+        mutate_setting = [False, False, False]
         payload = {
             "type": "script_function_payload",
-            "function": f"0x1::token::create_unlimited_collection_script",
+            "function": f"0x1::token::create_collection_script",
             "type_arguments": [],
             "arguments": [
                 name.encode("utf-8").hex(),
                 description.encode("utf-8").hex(),
                 uri.encode("utf-8").hex(),
+                str(U64_MAX),
```

Comment:
this feels really unconventional... let's poke @CapCap and @hariria and go with what the group consensus is?

Reply:
similar as above comment